### ISSUE TITLE
Move repositories to organization?

### ISSUE THREAD

**Issue by renatopp** [renatopp/behavior3js#19]
Cross posted in https://github.com/renatopp/behavior3editor/issues/22

> What do you guys think about moving all repositories (behavior3editor|behavior3js|behavior3py) to behavior3 organization? I already created the organization but I want to hear your opinions before migrating. (this would happen before the next release, +- 2/3 weeks)

> IMO, this makes the project looking more "professional" and less like a personal tool.

**Comment by renatopp** [comment]
If I hear no arguments against it, I'm going to move the repositories next week.

**Comment by renatopp** [comment]
All projects have been moved to:
- https://github.com/behavior3

I'm closing all issues here.

**Comment by linhaiwei123** [comment]
is there any different between the old release version and the moved version?

**Comment by renatopp** [comment]
@linhaiwei123 yes, there are some bug fixes (https://github.com/behavior3/behavior3js/commits/master)
